### PR TITLE
Update `add_toffolibox`

### PR DESCRIPTION
**Pull request by sjdilkes**
Also make sure docs cover ToffoliBox operation

### CODE REVIEW

File: tket/src/Circuit/include/Circuit/Boxes.hpp [hunk 1]
```diff
@@ -723,23 +730,55 @@ class ToffoliBox : public Box {
...
   Op_ptr symbol_substitution(
       const SymEngine::map_basic_basic &) const override {
     return Op_ptr();
   }
 
+  /** Get the Pauli string */
+  unsigned get_n_qubits() const { return n_qubits_; }
+
+  /** Get the phase parameter */
+
```
Comment (cqc-alec) [inline tket/src/Circuit/include/Circuit/Boxes.hpp:765]: Seems like the wrong docstring.

File: tket/src/Circuit/include/Circuit/Boxes.hpp [hunk 2]
```diff
@@ -723,23 +730,55 @@ class ToffoliBox : public Box {
...
+   * Copy constructor
+   */
+  ToffoliBox(const ToffoliBox &other);
+
   Op_ptr symbol_substitution(
       const SymEngine::map_basic_basic &) const override {
     return Op_ptr();
   }
 
+  /** Get the Pauli string */
```
Comment (cqc-alec) [inline tket/src/Circuit/include/Circuit/Boxes.hpp:761]: Wrong docstring?

File: pytket/binders/circuit/Circuit/add_op.cpp [hunk 3]
```diff
@@ -466,6 +466,19 @@ void init_circuit_add_op(py::class_<Circuit, std::shared_ptr<Circuit>> &c) {
...
           py::arg("pauliexpbox"), py::arg("qubits"))
+      .def(
+          "add_toffolibox",
+          [](Circuit *circ, const ToffoliBox &box, const qubit_vector_t &qubits,
+             const py::kwargs &kwargs) {
+            return add_box_method<UnitID>(
+                circ, std::make_shared<ToffoliBox>(box),
+                {qubits.begin(), qubits.end()}, kwargs);
+          },
+          "Append a :py:class:`ToffoliBox` to the "
```
Comment (cqc-alec) [inline pytket/binders/circuit/Circuit/add_op.cpp:477]: Can the line breaks be improved here, just for readability?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: tket/src/Circuit/include/Circuit/Boxes.hpp (hunk 1 above)
Comment (sjdilkes) [reply tket/src/Circuit/include/Circuit/Boxes.hpp]: Done

File: tket/src/Circuit/include/Circuit/Boxes.hpp (hunk 2 above)
Comment (sjdilkes) [reply tket/src/Circuit/include/Circuit/Boxes.hpp]: Done

File: pytket/binders/circuit/Circuit/add_op.cpp (hunk 3 above)
Comment (sjdilkes) [reply pytket/binders/circuit/Circuit/add_op.cpp:477]: As all Box operations in this file have similarly poorly formatted line breaks, suggestion is to update all of them in a separate PR.

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**